model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -60,10 +60,6 @@ message ArtifactMetadata {
   // (Optional) The Unix-like permissions of the artifact
   uint32 permissions = 2;
 
-  // (Optional) The base64-encoded md5 checksum of the artifact. Used, among other things, by
-  // harness boot code to validate the integrity of the artifact.
-  string md5X = 3;
```

Comment:
Do we need to `reserve 3;`? Or would we only do that for bugfix releases?

Reply:
I don't think it needs to be reserved; these protos are not persisted anywhere (yet?).